### DISCUSSION TITLE
关于文本翻译中显示翻译进度的功能

### DISCUSSION THREAD

**Discussion by already-love** [Ayanaminn/N46Whisper#29] [Ideas]
老师您好，我用N46Whisper在一个克罗地亚语的视频上尝试进行AI翻译，并且最后的输出栏在不断输出克罗地亚语和中文的文字。然而由于我根本不懂克罗地亚语，也看不懂原视频，所以我不能实时知道翻译进行到了视频的几分几秒，还有多久自动翻译才能完成。以后若能在colab输出的时候直接显示翻译进度的话，或许能更直观，尤其是对于陌生语种的长视频翻译。谢谢！

**Comment by Ayanaminn** [comment]
8e1d796ff28896a4b8704728bc6b730a2addc8d6
添加了这个功能，欢迎测试
